Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1515,20 +1515,29 @@ public RelativeVigorIndex RVI(Symbol symbol, int period, MovingAverageType movin
         /// Creates an RelativeDailyVolume indicator for the symbol. The indicator will be automatically
         /// updated on the given resolution.
         /// </summary>
-        /// <param name="symbol">The symbol whose RDV we want</param>
-        /// <param name="period">The period of the RDV</param>
+        /// <param name="symbol">The symbol whose RV we want</param>
+        /// <param name="period">The period of the RV</param>
         /// <param name="resolution">The resolution</param>
         /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to the Value property of BaseData (x => x.Value)</param>
         /// <returns>The Relative Volume indicator for the given parameters</returns>
-        public RelativeDailyVolume RDV(Symbol symbol, int period = 2, Resolution resolution = Resolution.Daily, Func<IBaseData, TradeBar> selector = null)
+        public RelativeDailyVolume RDV(Symbol symbol, int period = 2, Resolution resolution = Resolution.Hour, Func<IBaseData, TradeBar> selector = null)
         {
+            if (resolution == Resolution.Daily)
+            {
+                throw new ArgumentException("RelativeDailyVolume Indicator must have a resolution less than Resolution.Daily", nameof(resolution));
+            }
             var name = CreateIndicatorName(symbol, $"RDV({period})", resolution);
             var relativeDailyVolume = new RelativeDailyVolume(name, period);
             RegisterIndicator(symbol, relativeDailyVolume, resolution, selector);
+            var numberOfBars = (TimeSpan.FromDays(1) / resolution.ToTimeSpan()) * period;
 
             if (EnableAutomaticIndicatorWarmUp)
             {
-                WarmUpIndicator(symbol, relativeDailyVolume, resolution);
+                var history = History(symbol, (int)numberOfBars, resolution);
+                foreach(TradeBar tradeBar in history)
+                {
+                    relativeDailyVolume.Update(tradeBar);
+                }
```

Comment:
Shouldn't duplicate the purpose/responsibility of `WarmUpIndicator`, the _indicator_ should return a valid number of bars required to warmup it up

Reply:
Please see changes. Still passes locally but not on GitHub.